tests/test_auto_trace.py

```diff
@@ -73,34 +73,36 @@ def test_auto_trace_sample(exporter: TestExporter) -> None:
                 },
             },
             {
-                'name': 'Calling async_gen via @instrument (pending)',
+                'name': 'Calling tests.auto_trace_samples.foo.async_gen.<locals>.inner (pending)',
                 'context': {'trace_id': 1, 'span_id': 4, 'is_remote': False},
                 'parent': {'trace_id': 1, 'span_id': 3, 'is_remote': False},
                 'start_time': 2000000000,
                 'end_time': 2000000000,
                 'attributes': {
                     'code.filepath': 'foo.py',
                     'code.lineno': 123,
-                    'code.function': 'async_gen',
-                    'logfire.msg_template': 'Calling async_gen via @instrument',
+                    'code.function': 'async_gen.<locals>.inner',
+                    'logfire.msg_template': 'Calling tests.auto_trace_samples.foo.async_gen.<locals>.inner',
+                    'logfire.tags': ('testing', 'auto-tracing'),
                     'logfire.span_type': 'pending_span',
-                    'logfire.msg': 'Calling async_gen via @instrument',
+                    'logfire.msg': 'Calling tests.auto_trace_samples.foo.async_gen.<locals>.inner',
                     'logfire.pending_parent_id': '0000000000000001',
                 },
             },
             {
-                'name': 'Calling async_gen via @instrument',
+                'name': 'Calling tests.auto_trace_samples.foo.async_gen.<locals>.inner',
                 'context': {'trace_id': 1, 'span_id': 3, 'is_remote': False},
                 'parent': {'trace_id': 1, 'span_id': 1, 'is_remote': False},
                 'start_time': 2000000000,
                 'end_time': 3000000000,
                 'attributes': {
                     'code.filepath': 'foo.py',
                     'code.lineno': 123,
-                    'code.function': 'async_gen',
-                    'logfire.msg_template': 'Calling async_gen via @instrument',
+                    'code.function': 'async_gen.<locals>.inner',
+                    'logfire.msg_template': 'Calling tests.auto_trace_samples.foo.async_gen.<locals>.inner',
```

Comment:
Is there a reason the template isn't just something like `Calling {func}`?

Reply:
Auto-tracing has always included the fully qualified name. The name wasn't qualified in this test before because that function was instrumented with a hardcoded template.

If you mean 'why is the template the full message rather than something template-like' it's because auto-tracing is optimised to not go through message formatting, and these are still low cardinality enough for span names. Searching for `span_name = 'Calling {func}'` wouldn't be very useful. It's possible to filter on `otel_scope_name = 'logfire.auto_tracing'` instead.